src/components/NotesMindMap/NotesMindMapSagas.js

```diff
@@ -58,6 +61,21 @@ function* changeSelectedNote({ data: { note, edges } }) {
   }
 }
 
+function* handleSearchResultClick({ data: { note, edges } }) {
+  const targetNote = note;
+  if (didNotAttemptToFetchChildren(targetNote, edges)) {
```

Comment:
I don't think we need this `if()`.
It's used for `fetchChildNotes()`, to prevent refetching child notes when user clicks on a note which already has children fetched and displayed on the mind map.
But for search, we don't have such a situation.